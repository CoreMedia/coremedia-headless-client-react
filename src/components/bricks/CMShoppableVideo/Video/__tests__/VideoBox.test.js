// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import { VideoBox } from '..';

describe('VideoBox Component', () => {
  it('should render correctly with show = true', () => {
    const tree = renderer.create(<VideoBox show>Content</VideoBox>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should render correctly with show = false', () => {
    const tree = renderer.create(<VideoBox>Content</VideoBox>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
