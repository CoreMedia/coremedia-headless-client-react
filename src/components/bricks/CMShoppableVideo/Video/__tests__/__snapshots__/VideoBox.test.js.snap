// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoBox Component should render correctly with show = false 1`] = `
.c0 {
  opacity: 0;
  -webkit-transition: opacity 0.6s linear 0s;
  transition: opacity 0.6s linear 0s;
}

<div
  className="c0"
>
  Content
</div>
`;

exports[`VideoBox Component should render correctly with show = true 1`] = `
.c0 {
  opacity: 1;
  -webkit-transition: opacity 0.6s linear 0s;
  transition: opacity 0.6s linear 0s;
}

<div
  className="c0"
>
  Content
</div>
`;
